kornia/filters/kernels.py

```diff
@@ -191,7 +204,7 @@ def get_diff_kernel_3x3() -> Tensor:
     return tensor([[-0.0, 0.0, 0.0], [-1.0, 0.0, 1.0], [-0.0, 0.0, 0.0]])
 
 
-def get_diff_kernel3d(device=torch.device('cpu'), dtype=torch.float) -> Tensor:
+def get_diff_kernel3d(device: Device = torch.device('cpu'), dtype: torch.dtype = torch.float) -> Tensor:
```

Comment:
```suggestion
def get_diff_kernel3d(device: Device = None, dtype: Dtype = None) -> Tensor:
```
from kornia.core import Dtype